content/pt-br/docs/concepts/cluster-administration/flow-control.md

```diff
@@ -0,0 +1,630 @@
+
+Cada `kube-apiserver` mantém independentemente os requisitos obrigatórios e
+objetos de configuração sugeridos, usando comportamento inicial e periódico.
+Assim, em uma situação com uma mistura de servidores de diferentes versões
+pode haver _thrashing_ desde que servidores diferentes tenham
+opiniões sobre o conteúdo adequado desses objetos.
+
+Para os objetos de configuração obrigatórios, a manutenção consiste em
+garantir que o objeto existe e, se existir, tem a especificação adequada.
+O servidor se recusa a permitir uma criação ou atualização com uma especificação que é
+inconsistente com o comportamento do guarda-corpo do servidor.
+
+A manutenção de objetos de configuração sugeridos é projetada para permitir
+que suas especificações sejam substituídas. A exclusão, por outro lado, não é
+respeitada: a manutenção restaurará o objeto. Se você não quer um
+objeto de configuração sugerido, então você precisa mantê-lo por perto, mas defina
+sua especificação para ter consequências mínimas. Manutenção de objetos
+sugeridos também é projetada para suportar a migração automática quando uma nova
+versão do `kube-apiserver` é lançada, embora potencialmente com
+_thrashing_ enquanto há uma população mista de servidores.
+
+A manutenção de um objeto de configuração sugerido consiste em cria-lo
+--- com a especificação sugerida pelo servidor --- se o objeto não
+existir. OTOH, se o objeto já existir, o comportamento de manutenção
+depende se os `kube-apiservers` ou os usuários controlam o
+objeto. No primeiro caso, o servidor garante que a especificação do objeto
+é o que o servidor sugere; no último caso, a especificação é deixada
+sozinho.
+
+A questão de quem controla o objeto é respondida primeiro olhando
+para uma anotação com a chave `apf.kubernetes.io/autoupdate-spec`. Se
+existe tal anotação e seu valor é `true` então o
+kube-apiservers controlam o objeto. Se houver tal anotação
+e seu valor for `false`, os usuários controlarão o objeto. Se
+nenhuma dessas condições é satisfeita entaão a `metadata.generation` do
+objeto é consultado. Se for 1, o kube-apiservers controla
+o objeto. Caso contrário, os usuários controlam o objeto. Essas regras foram
+introduzido na versão 1.22 e sua consideração de
+`metadata.generation` é para migrar do mais simples
+comportamento anterior. Usuários que desejam controlar um objeto de configuração sugerido
+deve definir sua anotação `apf.kubernetes.io/autoupdate-spec`
+para 'falso'.
+
+A manutenção de um objeto de configuração obrigatório ou sugerido também
+inclui garantir que ele tenha uma anotação `apf.kubernetes.io/autoupdate-spec`
+que reflete com precisão se os kube-apiservers
+controlam o objeto.
+
+A manutenção também inclui a exclusão de objetos que não são obrigatórios
+nem sugeridos, mas são anotados
+`apf.kubernetes.io/autoupdate-spec=true`.
+
+## Isenção de simultaneidade da verificação de integridade
+
+A configuração sugerida não dá nenhum tratamento especial à saúde
```

Comment:
This sentence can be improved, like `tratamento especial a checagem de saúde das requisições...`